cluster-provision/gocli/cmd/run.go

```diff
@@ -812,7 +812,7 @@ func provisionK8sOptions(sshClient libssh.Client, k8sClient k8s.K8sDynamicClient
 	}
 
 	if n.CNAO {
-		cnaoOpt := cnao.NewCnaoOpt(k8sClient, sshClient)
+		cnaoOpt := cnao.NewCnaoOpt(k8sClient, sshClient, n.Multus)
```

Comment:
@aerosouund Can you please explain the reasoning to the code as well. This was something that took me time to connect. Please enhance the commit message as well

Reply:
indeed it can be enhanced, will do